### PR TITLE
[DO NOT MERGE] AF-593+: Align appformer and kie-soup version management with platform.

### PR DESCRIPTION
**Pull request by mbarkley**

### CODE REVIEW

**Comment by mbarkley** [comment]
Related:
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/630\
https://github.com/kiegroup/kie-soup/pull/18
https://github.com/kiegroup/appformer/pull/105
https://github.com/kiegroup/droolsjbpm-integration/pull/1297
https://github.com/kiegroup/kie-uberfire-extensions/pull/60
https://github.com/kiegroup/kie-wb-common/pull/1310
https://github.com/kiegroup/drools-wb/pull/721
https://github.com/kiegroup/jbpm-wb/pull/953
https://github.com/kiegroup/optaplanner-wb/pull/241
https://github.com/kiegroup/jbpm-designer/pull/709
https://github.com/kiegroup/kie-wb-distributions/pull/659

**Comment by mbarkley** [comment]
Related:
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/630\
https://github.com/kiegroup/kie-soup/pull/18
https://github.com/kiegroup/appformer/pull/105
https://github.com/kiegroup/droolsjbpm-integration/pull/1297
https://github.com/kiegroup/kie-uberfire-extensions/pull/60
https://github.com/kiegroup/kie-wb-common/pull/1310
https://github.com/kiegroup/drools-wb/pull/721
https://github.com/kiegroup/jbpm-wb/pull/953
https://github.com/kiegroup/optaplanner-wb/pull/241
https://github.com/kiegroup/jbpm-designer/pull/709
https://github.com/kiegroup/kie-wb-distributions/pull/659

**Comment by mbarkley** [comment]
Jenkins, please retest this.

**Comment by mbarkley** [comment]
Jenkins, please retest this.

**Comment by mbarkley** [comment]
Jenkins, please retest this.

**Comment by mbarkley** [comment]
Jenkins, please retest this.

File: kie-server-parent/kie-server-tests/pom.xml [hunk 1]
```diff
@@ -635,6 +637,11 @@
             <artifactId>jaxb-impl</artifactId>
             <version>${version.com.sun.xml.bind}</version>
           </dependency>
+          <dependency>
+            <groupId>org.apache.activemq</groupId>
+            <artifactId>artemis-jms-client</artifactId>
+            <version>${version.org.apache.activemq.wildfly10}</version>
+          </dependency>
```
Comment (psiroky) [inline kie-server-parent/kie-server-tests/pom.xml]: I am not sure I understand why is this change needed. Is it related to the uberfire->appformer rename?

Comment (mbarkley) [reply kie-server-parent/kie-server-tests/pom.xml]: When I consolidated the dependency management in this PR, I overrode the versions of several `org.apache.activemq` dependencies, but not all the ones used by some of the kie-server tests. Other dependencies had their versions managed from wildfly-parent.

TLDR: this change is necessary to fix missing method exceptions in some of the tests.

Comment (psiroky) [reply kie-server-parent/kie-server-tests/pom.xml]: OK, thanks for the explanation.

**Comment by mswiderski** [comment]
@mbarkley is this still valid or can it be closed?

**Comment by mbarkley** [comment]
@mswiderski Closed all of these for now, since they are so out of date.
